File: api/code-samples/javascript/klaytnDebug/blockchainInspection/startCollectingTrieStats.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const address = "0x0000000000000000000000000000000000000000"

    sdk.debug.startCollectingTrieStats(address, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/blockchainInspection/startCollectingTrieStats.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0x0000000000000000000000000000000000000000"

sdk = OpenSDK(host)
debug_response = sdk.debug.start_collecting_trie_stats(address)

print(debug_response)
